charts/music-notifications/templates/service.yaml

```
@@ -0,0 +1,22 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ include "music-notifications.fullname" . }}
+  labels:
+    {{- include "music-notifications.labels" . | nindent 4 }}
+  {{- with .Values.service.annotations }}
+  annotations:
+{{ toYaml . | indent 4 }}
+  {{- end }}
+spec:
+  type: {{ .Values.service.type }}
+  ports:
+    - port: {{ .Values.service.port }}
+      targetPort: 3000
+      {{- if eq .Values.service.type "NodePort" }}
+      nodePort: {{ default "3000" .Values.service.nodePort }}
+      {{- end }}
+      protocol: TCP
+      name: http
```

nit: indent level is not consistent with other resources above.

```suggestion
  - port: {{ .Values.service.port }}
    targetPort: 3000
    {{- if eq .Values.service.type "NodePort" }}
    nodePort: {{ default "3000" .Values.service.nodePort }}
    {{- end }}
    protocol: TCP
    name: http
```